doc/nn_serial.txt

```diff
@@ -0,0 +1,55 @@
+nn_serial(7)
```

Comment:
Yep, glad you caught that @wirebirdlabs .  I was speaking with the author about his plans for nn_serial. :-)